gulpTasks/injectVersion.js

```diff
@@ -0,0 +1,14 @@
+const gulp = require('gulp');
```

Comment:
This file is a modified version of the deleted file above. I used git to rename the file, but this is not being reflected by Github's diff.